pages/index.js

```diff
@@ -1,13 +1,32 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import styled from "styled-components";
+import axios from "axios";
 
-import LoginForm from "../src/components/login-form";
+import MemorizedLoginForm from "../src/components/login-form";
+import MemorizedVoteForm from "../src/components/vote-form";
 
 export default function Home() {
+  const [isLoggedIn, setLoginStatus] = useState(false);
+
+  const loginCheck = () => {
+    console.log("login check start");
+    if (isLoggedIn) {
+      console.log("dd");
+      return <MemorizedVoteForm />;
+    } else {
+      return (
+        <MemorizedLoginForm
+          isLoggedIn={isLoggedIn}
+          setLoginStatus={setLoginStatus}
+        />
+      );
+    }
+  };
+
   return (
     <Wrapper>
-      리액트 투-표
-      <LoginForm />
+      <Title>리액트 투-표</Title>
+      {loginCheck()}
```

Comment:
```suggestion
      {!isLoggedIn && (<LoginForm/>)}
      {isLoggedIn && (<VoteForm {...{setLoginStatus}}/>)}
```
1. 컴포넌트들은 ```return()``` 안에서 확인할 수 있게 하면 좋아요. 굳이 컴포넌트를 리턴하는 함수를 만들지 말고 위와 같이 리팩토링하면 더 좋을 것 같아요 !
2. ```console.log()```는 commit할 때 다 지워주시는 게 좋아요!
3. ```isLoggedIn```를 ```<MemorizedLoginForm/>```에 넘길 필요는 없을 것 같아요.
4. props 이름과 넘길 값을 담고 있는 변수의 이름이 같다면 spread operator를 통해 props를 넘기면 좋아요!
5. ```LoginForm, VoteForm```에 모두 React.memo를 적용했지만 그렇다고 컴포넌트들을 import해서 쓸 때 ```Memoized```를 붙여 import한 후 쓸 필요는 없어요. [참고](https://beomy.tistory.com/22)